clients/assessment_component/src/styles.css

```
@@ -0,0 +1,3 @@
+@import 'tailwindcss';
+@import '../../tailwind-base.css';
+@config '../tailwind.config.js';
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@config" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)